cmd/argo-watcher/server.go

```diff
@@ -13,7 +13,7 @@ import (
 func initLogs(logLevel string) {
 	if logLevel, err := zerolog.ParseLevel(logLevel); err != nil {
 		log.Warn().Msgf("Couldn't parse log level. Got the following error: %s", err)
```

Comment:
I missed this initially, but in this particular function we probably should set the log level first, the start emitting log messages

Reply:
We don't need to. We can even remove setting the log level to INFO as it's the default. I have removed it.